app/organisation/rest.py

```diff
@@ -324,6 +326,71 @@ def notify_users_of_request_to_go_live(service_id):
     return {}, 204
 
 
+@organisation_blueprint.route(
+    "/notify-org-member-about-continuation-of-go-live-request/<uuid:service_id>", methods=["POST"]
+)
+def notify_org_member_about_continuation_of_go_live_request(service_id):
+    data = request.get_json()
+    validate(data, post_notify_org_member_about_continuation_of_go_live_request)
+
+    template = dao_get_template_by_id(current_app.config["ORGANISATION_CONTINUE_GO_LIVE_REQUEST_TEMPLATE_ID"])
+    service = dao_fetch_service_by_id(service_id)
```

Comment:
[suggestion] Should we rename `service` to `subject_service` or `trial_mode_service`, or `go_live_service` or something, since we already also get and use GOV.UK Notify service in this view, and it's a bit confusing?

Similar in the other view.

Reply:
I'm fairly ambivalent on this. I think it's fairly standard for `service` to refer to the service that the request is primarily about. Calling it `go_live_service` might be more clear, but when working across the API if the subject of every API action had a different name I think I'd find that to be more mental overhead.

I maybe consider this to be a bit like `x` and `i` variables, where the non-descriptive name has some broader understanding behind it. But I also didn't think about it _that_ much...

Reply:
Ok, that's fair 👍🏼